tests/test_packages/test_skills/test_tac_control/test_handlers.py

```diff
@@ -481,7 +481,7 @@ def test_on_transaction_valid(self):
             performative=TacMessage.Performative.TRANSACTION_CONFIRMATION,
             to=tac_participant_counterparty,
             sender=self.skill.skill_context.agent_address,
-            target=counterparty_dialogue.last_message.message_id - 1,
+            target=counterparty_dialogue.last_message.message_id + 1,
```

Comment:
Hm I think this was correct, this is supposed to be the target.

Reply:
done